spring-cloud-app-broker-deployer/src/main/java/org/springframework/cloud/appbroker/deployer/GetApplicationRequest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2002-2019 the original author or authors
```

Comment:
Let's live in the present 😸.
Probably applies to all the license files. If you rebase master you should get a proper error because of the year.

Reply:
Thanks! Moving to present 🐈